script/zip_manifests/dist_zip.linux.arm.manifest

```diff
@@ -68,5 +68,6 @@ resources/default_app.asar
 snapshot_blob.bin
 swiftshader/libEGL.so
 swiftshader/libGLESv2.so
+swiftshader/libvk_swiftshader_icd.json
```

Comment:
The contents of this file is:

```
{
  "file_format_version": "1.0.0",
  "ICD": {
    "library_path": "..\\..\\bin\\vulkan\\x64\\Debug\\libvk_swiftshader.dll",
    "api_version": "1.0.5"
  }
}
```

given that the `../../bin` path isn't correct for distributed Electron, it seems like this file is probably unnecessary for us.